tox.ini

```diff
@@ -51,6 +52,11 @@ commands = flake8 src/ tests/ lib/
 deps =
     flake8==3.7.9
 
+[testenv:woke]
```

Comment:
Do you think this should be one of the checks in `lint` rather than a separate check? It seems to share many characteristics with a linting tool

Reply:
I agree it shares some characteristics, but it feels different enough to me that it warrants a different target. Happy to be overridden if I'm in the minority on that.

Reply:
Maybe it will be appropriate to add this target the `envlist`, also, `lint`is missing